tests_integration_with_local_chain/tools/test_cow_order.py

```
@@ -7,13 +7,30 @@
     WrappedxDaiContract,
     sDaiContract,
 )
+from prediction_market_agent_tooling.tools.cow.cow_order import (
+    get_buy_token_amount,
+    swap_tokens_waiting,
+)
+from prediction_market_agent_tooling.tools.web3_utils import xdai_to_wei
 from prediction_market_agent_tooling.tools.cow.cow_order import swap_tokens_waiting
```

Ruff (0.8.2)

Redefinition of unused `swap_tokens_waiting` from line 12

Remove definition: `swap_tokens_waiting`

(F811)